api/v1alpha1/cloudstorage_types.go

```diff
@@ -1,23 +1,26 @@
+/*
+Copyright 2021.
```

Comment:
do you know where 2021 is coming from?

Reply:
https://github.com/openshift/oadp-operator/blob/master/hack/boilerplate.go.txt

Reply:
should we update that here?

Reply:
since it is not necessary for it to work, no. Can be a separate PR